helm/hpcc/values.schema.json

```diff
@@ -316,6 +316,15 @@
         "visibilities": {
           "type": "object",
           "description" : "Global visibilities configuration",
+          "properties" : {
+            "exposedServices": {
+              "type": "array",
+              "items": { "type": "string" }
+	    },
```

Comment:
minor: indentation issue, and lines below.

Reply:
fixed